main.cpp

```diff
@@ -204,6 +437,50 @@ Thing 10) Laptop Computer
     2) Send an email
     3) Play music
  */
+struct Laptop
+{
+    //Keyboard
+    unsigned int keyboard = 1;
+    //Screen
+    unsigned int screen = 1080;
+    //Webcam
+    bool hasWebcam = false;
+    //TouchPad
+    float touchpadSensitivity = 3.33f;
+    //Speakers
+    unsigned int numOfSpeakersLaptop = 2;
+
+    struct LaptopSpeaker
+    {
+        //Manufacturer Model
+        std::string modelOfSpeaker = "Bose3";
+        //Warranty?
+        bool hasWarranty = false;
+        //Year Made
+        int yearMade = 2020;
+        //Color of woofer
+        std::string color = "blue";
+        //Default volume
+        float defaultVolume = 0.55f;
+
+        // Ok to play sounds?
+        void playSounds ( bool muteEnabled = false);
+        // Increase the volume by
+        void increaseVolume ( float amountToIncrease = 0.5f); // out of 10
+        // Sleep timer time remaining
+        void sleepTimer ( unsigned int timeUntilMute = 600); //seconds
+    };
+    // Play fav song on speakers
+    void playFavoriteSong ( LaptopSpeaker playSound );
+    // Length of time remaining for song
+    int howLongToPlay ( LaptopSpeaker songDuration );
+    // stop powering speakers when computer turned off
+    void turnOffOnShutdown ( LaptopSpeaker powerDown );
+
```

Comment:
So, do all of these functions not use the `LaptopSpeaker` that is a member variable?  They interact with some other `LaptopSpeaker`?  You would think that a laptop would use its own speakers... (`speakersInLaptop`)
- [ ] revise